src/Microsoft.Framework.PackageManager/Utils/FileOperationUtils.cs

```diff
@@ -38,12 +37,6 @@ private static void DeleteRecursive(string deletePath)
 
         public static bool MarkExecutable(string file)
         {
-            if (PlatformHelper.IsWindows)
```

Comment:
Definitely safer and results in less code duplication. I'd also have to change #1931 if this went in first :smiling_imp: